writer.go

```diff
@@ -185,6 +185,9 @@ type Writer struct {
 	// If nil, DefaultTransport is used.
 	Transport RoundTripper
 
+	// AllowAutoTopicCreation notifies writer to create topic is missing.
+	AllowAutoTopicCreation bool
```

Comment:
Any reason to not introduce this in the writer config too?
https://github.com/segmentio/kafka-go/issues/1278

If I am using WriterConfig to create the writer, is it safe to just set this config after calling `NewWriter`?

Reply:
@erikdw @achille-roussel Here's a PR in case 1278 is a desirable feature request: https://github.com/segmentio/kafka-go/pull/1279 if